packages/mobile/src/config.ts

```diff
@@ -71,6 +77,7 @@ export const DEFAULT_SYNC_MODE: GethSyncMode = Config.DEFAULT_SYNC_MODE
   : GethSyncMode.Lightest
 // TODO Remove when feature is stable
 export const USE_PHONE_NUMBER_PRIVACY = true
+export const SHOW_CASH_OUT = stringToBoolean(Config.SHOW_CASH_OUT || 'false')
```

Comment:
if `Config.SHOW_CASH_OUT` is a string then  `Config.SHOW_CASH_OUT || 'false'` wont the false here not ever be needed?  i guess it would for the empty string state?

Reply:
Yeah, if `Config.SHOW_CASH_OUT` is an empty string or undefined (which should be the case for environments in which we do not want this feature) this would evaluate to `false`.